components/board/commonsysfs/ioctl.go

```diff
@@ -0,0 +1,168 @@
+package commonsysfs
```

Comment:
this needs a go build tag guard for linux only. otherwise this wont compile on other systems when included